src/docs/asciidoc/jet-concepts.adoc

```diff
@@ -133,7 +133,7 @@ tokenizing processor:
 image::wordcount-tokenizer.png[Word-counting DAG with tokenizer vertex parallelized,600,170,align="center"]
 
 The input processor can now use all the available tokenizers as a pool
-and submit to any one whose queue has some room.
+and submit to any whose queue has some room.
```

Comment:
Discussed with a colleague, the original was actually correct. It stresses that the processor submits to one of the queues as opposed to any number of queues that have room.